react/test/card.js

```diff
@@ -0,0 +1,25 @@
+// test/components/Card.js
+import Card from '../src/components/Card';
+import { mount } from 'enzyme';
+import React from 'react';
+
+describe('Component: Card', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    jasmineEnzyme();
+    wrapper = mount(
+      <Card/>
+    );
+  });
+
+  it('will pass', () =>{
```

Comment:
I would like to see an actual react test for your react code :) Timebox this to maybe 15-20 minutes, and if you're still struggling just move on.